authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,186 @@
+
+Both modes of communication - crosslinks and ICMP - are asynchronous and thus non-instant. The appearance of real-time communication will be a matter of userland implementation.
+
+## Node types
+
+In Ethereum, the following economy-user and node roles (client software types) exist:
+
+|Name       |Role                                                  |Incentivized|Optional|
+|-----------|------------------------------------------------------|------------|--------|
+|Beacon node|Full node of all shards. Passes on data to validators.|No          |No      |
+|Validator  |Proposes and attests to blocks based on beacon data.  |Yes         |No      |
+|VDF ASIC   |Enhances RANDAO to provide better randomness.         |No          |Yes     |
+
+Beacon nodes can potentially be incentivized by being archive nodes of all shards, which means they keep even the data which was wiped due to state rent expiration. A state-rent wiped contract can be restored by an archive node, for a fee. Archive nodes as-a-service are probable in Ethereum 2.0.
+
+In Polkadot:
+
+|Name       |Role                                                        |Incentivized|Optional |
+|-----------|------------------------------------------------------------|------------|---------|
+|Collator   |Full node of particular shard. Passes on data to validators.|Possibly    |No       |
+|Validator  |Full client of all parachains and parathreads. Proposes and attests to blocks based on collator data.      |Yes         |No       |
+|Nominator  |Nominates a validator through which to proxy-stake.         |Yes         |In theory|
+|Fishermen  |Monitor the network for misdeeds like equivocation or illegal randomness.|Yes|Yes|
+
+Validators can be fishermen, which makes fishermen optional. Nominators are _theoretically_ unnecessary, i.e. the network would work fine without them if the validators had enough self-stake to keep themselves in the validator pool.
+
+## Inflation
+
+In both Polkadot and Ethereum the inflation is dynamic and defined as "minimum necessary inflation". The exact number will depend on the amount of staked coins.
+
+## Validators and staking economy
+
+### Polkadot
+
+In Polkadot a validator is a full client (>>FULL OR LIGHT?) of all chains and threads, thus very resource intensive on hard drive space and read/write ops, and on bandwidth.
```

Comment:
The validator should only be doing light client verification on the parachains so this would mean it would act as a light client. AFAIK this is how it works, but probably good to check with a researcher if there's situations where validators may need to sync parachain data. For some chains this wouldn't even be possible, such as with private chains where the data is kept secret by the collators.

Reply:
Indeed, Alistair confirms it's always light client.